Makefile

```diff
@@ -49,8 +58,13 @@ else
 	docker run --rm -v $(shell pwd):/app -w /app $(GOLANGCILINT_IMG) $(GOLANGCILINT) $(GOLANGCILINT_ARGS)
 endif
 
+clean: clean-fixtures clean-cov
```

Comment:
I'm not sure we want to `clean-fixtures` with `make clean`, these take a long time to generate again (hour+) and are git controlled for GHA.

Reply:
no problemo, ill drop it